RoosterPlanner.Api/Controllers/ParticipationsController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using RoosterPlanner.Api.Models;
+using RoosterPlanner.Common;
+using RoosterPlanner.Models;
+using RoosterPlanner.Service;
+using RoosterPlanner.Service.DataModels;
+
+namespace RoosterPlanner.Api.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ParticipationController : ControllerBase
+    {
+        private readonly ILogger logger;
+        private readonly IParticipationService participationService;
+
+        public ParticipationController(ILogger logger, IParticipationService participationService)
+        {
+            this.logger = logger;
+            this.participationService = participationService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> GetUserParticipation(Guid personGuid)
+        {
+            try
+            {
+                if (personGuid == Guid.Empty) return BadRequest("No valid id.");
+                TaskListResult<Participation> result = await participationService.GetUserParticipations(personGuid);
+
+                if (!result.Succeeded) return UnprocessableEntity();
+
+                List<ParticipationViewModel> participationViewModels = result.Data
+                    .Select(ParticipationViewModel.CreateVm)
+                    .ToList();
+
+                return Ok(participationViewModels);
+            }
+            catch (Exception ex)
+            {
+                logger.Error(ex, "ParticipationController: Error occured.");
+                Response.Headers.Add("message", ex.Message);
+            }
+
+            return NoContent();
```

Comment:
Misschien om even te kijken welke Http Status codes je wilt gebruiken. Misschien een paar richtlijnen opstellen:
Bijv. in een catch altijd een 422 UnprocessableEntity met een eigen error object en een 404 NotFound bij methoden waarbij een enkel item wordt opgevraagd. (NoContent zou kunnen bij een methode die alleen iets wegschrijft en geen resultaat hoeft terug te geven, maar meestal is feedback gewenst)